frontend/src/components/taxons/TaxonsListContent.tsx

```diff
@@ -1,23 +1,52 @@
-import React from 'react'
-import { Taxon } from '../../models/frontend_models'
-import PageContainer from '../PageContainer'
+import React, { useEffect, useState } from 'react'
+import { Taxon, getAllItems } from '../../models/frontend_models'
 import AppPageHeader from '../AppPageHeader'
 import PageContent from '../PageContent'
+import AddButton from '../AddButton'
+import { Table } from 'antd'
+import { ObjectWithTaxon, getColumnsForTaxon } from '../shared/DefinitionsTable/TaxonTableColumns'
+import { IdentifiedTableColumnType } from '../shared/WorkflowSamplesTable/SampleTableColumns'
+import { useAppSelector } from '../../hooks'
+import { selectTaxonsByID } from '../../selectors'
 
 export interface TaxonsListContentProps {
-	taxons: Taxon[]
+	taxons: Taxon[],
 }
 
-function TaxonsListContent({taxons} : TaxonsListContentProps) {
+function TaxonsListContent() {
+	const taxonsState = useAppSelector(selectTaxonsByID)
+	const taxons: Taxon[] = getAllItems(taxonsState)
+	const [taxonColumns, setTaxonColumns] = useState<ObjectWithTaxon[]>();
```

Comment:
These aren't columns, they are data objects, so I would use `const [taxonsData, setTaxonsData]` here. 
I might be a good idea to move `const taxons: Taxon[] = getAllItems(taxonsState)` into the useEffect, since that variable is only used inside the useEffect. You would need to change the useEffect dependency to `[taxonsState]` (or `[taxonsByID]`) in that case.